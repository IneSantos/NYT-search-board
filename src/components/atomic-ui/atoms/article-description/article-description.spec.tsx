import React from "react";
import { render } from "@testing-library/react";
import ArticleDescription from "./article-description";

describe("ArticleDescription atom tests", () => {
  test(`ArticleDescription component render`, () => {
    const { container } = render(
      <ArticleDescription description="This is a description" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
