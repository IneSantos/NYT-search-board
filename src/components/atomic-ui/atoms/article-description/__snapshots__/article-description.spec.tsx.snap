// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleDescription atom tests ArticleDescription component render 1`] = `
<div
  class="description"
>
  This is a description
</div>
`;
